tests/util.go

```diff
@@ -115,3 +123,100 @@ func writeJson(jsonFile string, json string) (string, error) {
 	}
 	return jsonFile, nil
 }
+
+// ReplaceStringInFile replace the old string by the new one.
+// If n < 0, there is no limit on the number of replacements.
+func ReplaceStringInFile(file, old, new string, n int) error {
+	input, err := ioutil.ReadFile(file)
+	if err != nil {
+		return fmt.Errorf("Error while reading file %s: ", file)
+	}
+
+	output := bytes.Replace(input, []byte(old), []byte(new), n)
+
+	if err = ioutil.WriteFile(file, output, 0644); err != nil {
+		return fmt.Errorf("Error while writing file %s", file)
+	}
+	return nil
+}
+
+// DownloadFromUrl reads content from a url to a local file.
+func DownloadFromUrl(url, dest string) error {
+	output, err := os.Create(dest)
+	if err != nil {
+		return fmt.Errorf("Error while creating file %s", dest)
+	}
+	defer output.Close()
+
+	response, err := http.Get(url)
+	if err != nil {
+		return fmt.Errorf("Error while downloading %s", url)
+	}
+	defer response.Body.Close()
+
+	_, err = io.Copy(output, response.Body)
+	if err != nil {
+		return fmt.Errorf("Error while writing to %s", dest)
+	}
+	return nil
+}
+
+// RunVirtctlCommand runs virtctl command with specified args.
+func RunVirtctlCommand(args ...string) (string, error) {
```

Comment:
dont we already have this functionality in kubevirt repo?

Reply:
It's not merged, so I'd like it to be here.